docs/en/unloading/Spark_connector.md

```diff
@@ -218,6 +219,7 @@ The following parameters apply only to the Spark RDD reading method.
 | STRING              | DataTypes.StringType      |
 | DATE                | DataTypes.DateType        |
 | DATETIME            | DataTypes.TimestampType   |
+| JSON | DataTypes.StringType <br /> **NOTE:** <br /> **Supported since version 1.1.2**, and require StarRocks at least 2.5.13, 3.0.3, 3.1 or later |
```

Comment:
Supported since version 1.1.2**, and require StarRocks at least 2.5.13, 3.0.3, 3.1 or later
-> This data type is supported since Spark connector v1.1.2, and requires a StarRocks version of at least 2.5.13, 3.0.3, 3.1 or later.